tests/scripts/thread-cert/border_router/README.md

```diff
@@ -0,0 +1,23 @@
+# OpenThread Border Router Tests
+
+## Run Border Router (BR) tests locally
+
+BR tests run in isolated Docker network and containers, so new OTBR Docker image needs to be created before running the tests:
+
+```shell
+# Clear current OpenThread directory (remember to add new source files).
+git clean -xfd
+
+# Rebuild the OTBR Docker image if OTBR source code is updated.
+./script/test build_otbr_docker
+
+./script/test get_thread_wireshark
```

Comment:
This step is only needed when /tmp is cleaned. This script will delete and re-download the library if you re-run it, which is unnecessary.